src/main/java/dao/JanggiGameDao.java

```diff
@@ -0,0 +1,69 @@
+package dao;
+
+import domain.entity.JanggiGameEntity;
+import domain.game.Turn;
+import domain.piece.Team;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Optional;
+
+public class JanggiGameDao {
+
+    public void createTable(Connection connection) throws SQLException {
+        final var createTableQuery = """
+                CREATE TABLE IF NOT EXISTS janggi_game (
+                    id BIGINT PRIMARY KEY AUTO_INCREMENT,
+                    turn VARCHAR(20)
+                );
+                """;
+        try (final var statement = connection.createStatement()) {
+            statement.execute(createTableQuery);
+        }
+    }
+
+    public Long create(Connection connection, JanggiGameEntity entity) throws SQLException {
+        final var createQuery = """
+                INSERT INTO janggi_game (turn) VALUES (?)
+                """;
+        try (final var preparedStatement = connection.prepareStatement(createQuery, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, entity.getTurn().getTeam().name());
+            preparedStatement.executeUpdate();
+            try (final var generatedKeys = preparedStatement.getGeneratedKeys()) {
+                if (generatedKeys.next()) {
+                    return generatedKeys.getLong(1);
+                }
+                throw new SQLException("[ERROR] id 생성을 실패하였습니다");
+            }
+        }
+    }
+
+    public void update(Connection connection, Long janggiGameId, JanggiGameEntity entity) throws SQLException{
+        final var updateQuery = """
+                UPDATE janggi_game SET turn = ? WHERE id = ?;
+                """;
+        try (final var preparedStatement = connection.prepareStatement(updateQuery)) {
+            preparedStatement.setString(1, entity.getTurn().getTeam().name());
+            preparedStatement.setLong(2, janggiGameId);
+            preparedStatement.executeUpdate();
```

Comment:
executeUpdate()의 반환값을 체크해보면 어떨까요? 😃  (반환 결과가 어떤 값인지 먼저 알아보신 뒤)
이 값을 통해서 적절히 예외를 던져주면 업데이트가 실패했음을 감지할 수 있을 것 같아요. 🤔

```suggestion
            int rowsUpdated = preparedStatement.executeUpdate();
            if (rowsUpdated == 0) {
                throw new SQLException("No rows updated, possible invalid janggiGameId: " + janggiGameId);
            }
}
```

Reply:
생성, 수정, 삭제와 같은 DML문을 실행할 경우 변경되는 쿼리의 개수를 반환한다고 합니다!
만약 DDL일 경우 0을 반환한다고 해요!

그렇기에 생성, 수정이 되지 않는다면 0을 반환할 것이고 예외를 발생시키는 것이 좋을 것 같습니다!
전혀 인지하고 있던 부분 짚어주셔서 감사합니다! 수정하였습니다!

d23125a4fbe3b43eaf55a5c6b5c44ca992ccf163

Reply:
아..! DDL을 생각하지 못했네요. 👀
이 부분은 잘 확인해주신 것 같습니다. 🙏